src/Grafana.OpenTelemetry.Base/Instrumentations/CassandraInitializer.cs

```diff
@@ -19,5 +20,13 @@ protected override void InitializeMetrics(MeterProviderBuilder builder)
                 "AddCassandraInstrumentation",
                 new object[] { builder });
         }
+
+        protected override void InitializeTracing(TracerProviderBuilder builder)
+        {
+            // TODO Update upstream to use CassandraCSharpDriver.OpenTelemetry and
+            // the this can be changed to .AddSource(CassandraActivitySourceHelper.ActivitySourceName)
+            // and then also the user doesn't need to manually install the package.

```

Comment:
```suggestion
            // TODO Update upstream to use CassandraCSharpDriver.OpenTelemetry and
            // the this can be changed to .AddSource(CassandraActivitySourceHelper.ActivitySourceName)
            // and then also the user doesn't need to manually install the package.
            // See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2939.
```